cli/azd/pkg/project/importer.go

```diff
@@ -191,6 +198,10 @@ func (im *ImportManager) SynthAllInfrastructure(ctx context.Context, projectConf
 		}
 	}
 
+	if im.dotNetImporter.alphaFeatureManager.IsEnabled(alpha.FeatureId(alpha.Compose)) && len(projectConfig.Resources) > 0 {
+		return infraFsForProject(ctx, projectConfig)
+	}
+
 	return nil, fmt.Errorf("this project does not contain any infrastructure to synthesize")
```

Comment:
If the alpha-feature is not enabled, are we printing this error message about there's not infra to synth?

Reply:
We can definitely consider a better error message for enablement.